blazor/range-slider/how-to/slider-inside-dialog.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: Range Slider inside Dialog popup in Blazor Range Slider Component | Syncfusion
+description: Checkout and learn here all about Slider inside Dialog popup in Syncfusion Blazor Range Slider component and more.
+platform: Blazor
+control: Range Slider
+documentation: ug
+---
+
+# Range Slider Inside Dialog Popup in Blazor Range Slider Component
+
```

Comment:
Render Blazor Range Slider Inside Dialog Popup -- this heading is enough don't repeat component name @PrasanthMadhaiyan

Reply:
Done.